lefthook.yaml

```
@@ -30,15 +30,15 @@ pre-commit:
-        set -e
-        pnpm check_tests   
+      run: pnpm check_type_errors       
     9_generate_docs:
       run: |
         set -e
         pnpm generate:docs
         git add ./docs/docs/docs/auto-schema
+    10_check_tests:
+      run: |
+        set -e
+        pnpm check_tests 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)